TypeCobol.Test/SemanticDomain/ExpanderRefines.cbl

```diff
@@ -0,0 +1,17 @@
+﻿       IDENTIFICATION DIVISION.
```

Comment:
Typo in file name : ExpanderRefines instead of ExpanderRedefines.

Reply:
ok